src/main/java/com/incentives/piggyback/notification/controller/NotificationController.java

```diff
@@ -1,21 +1,42 @@
 package com.incentives.piggyback.notification.controller;
 
-import com.incentives.piggyback.notification.publisher.NotificationEventPublisher;
-import com.incentives.piggyback.notification.util.CommonUtility;
-import com.incentives.piggyback.notification.util.constants.Constant;
+import java.util.Calendar;
+
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.Calendar;
+import com.incentives.piggyback.notification.entity.BroadcastRequest;
+import com.incentives.piggyback.notification.exception.PiggyException;
+import com.incentives.piggyback.notification.publisher.NotificationEventPublisher;
+import com.incentives.piggyback.notification.service.NotificationService;
+import com.incentives.piggyback.notification.util.constants.Constant;
+import com.incentives.piggyback.notification.utils.CommonUtility;
+import com.incentives.piggyback.notification.utils.RestResponse;
+import com.incentives.piggyback.notification.utils.RestUtils;
 
 @RestController
+@RequestMapping(value="/notification")
 public class NotificationController {
+	
+	@Autowired
+	private NotificationService notificationService;
+	
 
     @Autowired
     private NotificationEventPublisher.PubsubOutboundGateway messagingGateway;
 
+	@RequestMapping(value = "/broadcast", method = RequestMethod.POST)
+	public ResponseEntity<RestResponse<String>> broadcastNotification(@RequestBody 
+			BroadcastRequest broadcastRequest) throws PiggyException {
```

Comment:
Should push notification created event to pub/sub. Added a new story to take care of that.